### PR TITLE
Various changes, general maintenance…

### PR DESCRIPTION
**Pull request by radex**

### CODE REVIEW

**Comment by radex** [comment]
@rozPierog Delegating this to you again, because the most interesting stuff is in GitHub actions setup. I managed to get the times for iOS down to 7 minutes and Android down to 11 by using @actions/cache for Yarn and CocoaPods & ccache to speed up Objc/C++ compilation. Not quite as good as self-hosted, but should be good enough for the foreseeable future.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,34 +25,41 @@ jobs:
...
+    - name: 'pod install'
+      if: steps.pods-cache.outputs.cache-hit != 'true'
+      run: cd native/iosTest && bundle exec pod install
     - run: yarn test:ios
     - run: yarn swiftlint
   android:
     # https://github.com/ReactiveCircus/android-emulator-runner/issues/46
     runs-on: macos-10.15
-    name: Check Android
+    name: Android tests
```
Comment (rozPierog) [inline .github/workflows/ci.yml:62]: You can get Android to build even faster with caching - take Compose Sample app as a blueprint https://github.com/android/compose-samples/actions/runs/27015993/workflow

File: docs-master/Advanced/ProTips.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 
 ## Database viewer
 
-See this discussion: https://github.com/Nozbe/WatermelonDB/issues/710
+[See discussion](https://github.com/Nozbe/WatermelonDB/issues/710)
 
-**Android** - you can use the new database inspector which comes with the Android Studio Beta. https://medium.com/androiddevelopers/database-inspector-9e91aa265316  .
+**Android** - you can use the [new database inspector](https://medium.com/androiddevelopers/database-inspector-9e91aa265316) which comes with the Android Studio Beta.
```
Comment (rozPierog) [inline docs-master/Advanced/ProTips.md:8]: Database inspector is no longer only in beta branch. (also on newer versions Android Studio 4.2+ it's called `App Inspector` as it allows much more than only checking database)

**Review by rozPierog — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (radex) [reply .github/workflows/ci.yml:62]: noted for the future

**Pull request merged**